### ISSUE TITLE
Qgis crash

### ISSUE THREAD

**Issue by Lindeextreme** [pyproj4/pyproj#3]
I execute the following code lines in the Qgis python shell, then qgis crashes. When i execute the code outside of Qgis everthing works perfekt.
## Code

from pyproj import Proj, transform
in_projection = Proj(init="epsg:3857")
out_projection = Proj(init="epsg:4326")
lat, lng = transform(in_projection, out_projection, 887590, 7210569)

The enviroment:
OS: Ubuntu 14.04 LTS
Qgis: 2.6.1-Brighton
Project cbs: WGS 84 / Pseudo Mercator (EPSG:3857)
PROJ.4-Version: 480
Qt: 4.8.6
Python: 2.7.6
pyproj: 1.9.4

**Comment by micahcochran** [comment]
I tried this on the current version of QGIS 2.10.1-Pisa (OSGeo4W install) under Windows 7 Pro.  It worked fine.   It gave me these results:
```python
print lat, lng
7.97335663032 54.2128392776
```

Then I tried it on Ubuntu 15.04 with QGIS 2.10.1-Pisa.  QGIS crashed.  I even tried this:
```python
try:
   lat, lng = transform(in_projection, out_projection, 887590, 7210569)
except:
   print "Unexpected error:", sys.exc_info()[0]
   raise
```

and that didn't help.  The resulting error message when running QGIS from the prompt:
```
Segmentation fault (core dumped)
```

I think this is an issue with the QGIS software, not with pyproj.
